core/shared/src/main/scala/org/http4s/package.scala

```diff
@@ -79,4 +83,40 @@ package object http4s {
   /** A stream of server-sent events */
   type EventStream[F[_]] = Stream[F, ServerSentEvent]
 
+  // Syntax to enable JS-cross compilation without splitting files
+  // Nobody will actually be able to run these methods b/c they will fail to link
+
+  implicit private[http4s] class Fs2IoPathCompanionOps(p: fs2.io.file.Path.type) {
+    @nowarn212("cat=unused")
+    def fromNioPath(path: java.nio.file.Path): fs2.io.file.Path =
+      throw new UnsupportedOperationException
+  }
+
+  implicit private[http4s] class Fs2IoPathOps(p: fs2.io.file.Path) {
+    def toNioPath: java.nio.file.Path = throw new UnsupportedOperationException
+  }
+
+  implicit private[http4s] class Ip4sIpv4AddressCompanionOps(ip: ip4s.Ipv4Address.type) {
+    @nowarn212("cat=unused")
+    def fromInet4Address(a: java.net.Inet4Address): ip4s.Ipv4Address =
+      throw new UnsupportedOperationException
+  }
+
+  implicit private[http4s] class Ip4sIpv4AddressOps(ip: ip4s.Ipv4Address) {
+    def toInetAddress: java.net.Inet4Address = throw new UnsupportedOperationException
+  }
+
+  implicit private[http4s] class Ip4sIpv6AddressCompanionOps(ip: ip4s.Ipv6Address.type) {
+    @nowarn212("cat=unused")
+    def fromInet6Address(a: java.net.Inet6Address): ip4s.Ipv6Address =
+      throw new UnsupportedOperationException
+  }
+
+  implicit private[http4s] class Ip4sIpv6AddressOps(ip: ip4s.Ipv6Address) {
+    def toInetAddress: java.net.Inet6Address = throw new UnsupportedOperationException
+  }
+
+  implicit private[http4s] class Ip4sIpAddressOps(ip: ip4s.IpAddress) {
+    def toInetAddress: java.net.InetAddress = throw new UnsupportedOperationException
+  }
```

Comment:
Hasn't this accidentally seeped out from 0.23?

Reply:
Seems like it, we remove it here.